src/tracing/types.rs

```diff
@@ -114,6 +130,25 @@ impl CallTrace {
     }
 }
 
+/// A log with optional decoded data.
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct CallLog {
+    /// The raw log data.
+    pub raw_log: LogData,
+    /// The decoded event name.
+    pub decoded_name: Option<String>,
+    /// The decoded log parameters.
+    pub decoded_params: Option<Vec<(String, String)>>,
```

Comment:
what are the tuples here?
can we document it?

Reply:
Done